src/lsp/client/client.ts

```diff
@@ -24,7 +24,7 @@ export function activate(
 ) {
```

Comment:
This file is renamed from `extension.ts` to `client.ts`. `Extension.ts` should be used for the main entry point in an extension. So following the same spirit as `lsp/server` I named it `client.ts`. I thought of changing `activate()` too since it could be anything if it is not an extension. Besides, it's taking extra parameters in `updateDiagnosticSettingCommand` and `diagnosticSettingSection` that aren't expected by extension loader. But changing the file name suffices so I left `activate()` as is.